{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--527e3248-a2df-4a03-914c-6dbe612dcf40",
    "created": "2024-08-13T16:12:21.393262Z",
    "modified": "2024-08-13T16:12:21.393267Z",
    "name": "IBM Robotic Process Automation privilege escalation",
    "description": "\nIBM Robotic Process Automation 21.0.0 through 21.0.7.1 and 23.0.0 through 23.0.1 is vulnerable to incorrect privilege assignment when importing users from an LDAP directory.  IBM X-Force ID:  262481.\n\n",
    "external_references": [
        {
            "source_name": "cve",
            "external_id": "CVE-2023-38734"
        }
    ],
    "metrics": [
        {
            "cvssV3_1": {
                "attackComplexity": "HIGH",
                "attackVector": "NETWORK",
                "availabilityImpact": "HIGH",
                "baseScore": 6.6,
                "baseSeverity": "MEDIUM",
                "confidentialityImpact": "HIGH",
                "integrityImpact": "HIGH",
                "privilegesRequired": "HIGH",
                "scope": "UNCHANGED",
                "userInteraction": "NONE",
                "vectorString": "CVSS:3.1/AV:N/AC:H/PR:H/UI:N/S:U/C:H/I:H/A:H",
                "version": "3.1"
            },
            "format": "CVSS",
            "scenarios": [
                {
                    "lang": "en",
                    "value": "GENERAL"
                }
            ]
        }
    ],
    "references": [
        {
            "tags": [
                "vendor-advisory"
            ],
            "url": "https://www.ibm.com/support/pages/node/7028227"
        },
        {
            "tags": [
                "vdb-entry"
            ],
            "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/262481"
        }
    ]
}